### ISSUE TITLE
Add auth domain for blockstack apps

### ISSUE THREAD

**Issue by friedger** [blockstack/app.co#476]
**Is your feature request related to a problem? Please describe.**
Currently, there is no shared key between app.co/blockstack/apps and theblockstats.com

**Describe the solution you'd like**
Require app publishers to enter a auth domain used by their app (if they use blockstack auth)

**Describe alternatives you've considered**
the app.co appid could be added in the data set of theblockstats.com

**Additional context**
https://github.com/vrepsys/blockstats-page/issues/2

**Comment by GInxh** [comment]
Thank's friedgar. Sending your stuff over to app.co team for feedback.

**Comment by markmhx** [comment]
@friedger To be clear, is the desired result here that apps can display a link to blockstats.com from their profiles on App.co? Or something else?

**Comment by friedger** [comment]
In the first place, this should help to identify apps.
Then, it should make the app meta data reusable on other sites like blockstats.

Not sure which apps you are referring to. A listed app on app.co knows were it is listed on app.co and its auth domain, so they don't need this.

Related to the issue about app meta data ownership

**Comment by markmhx** [comment]
Ah I see, so it sounds like the idea is more to port app data from App.co to _other_ sites like blockstats, not so much the other way around. And the "app domain" (e.g. https://app.graphitedocs.com/) would assist in this case (and perhaps others) since blockstats uses that specific domain, and not the marketing one (e.g. https://www.graphitedocs.com/), as the distinct ID for each app (since the app-domain value shows up in the profiles of users publicly).

We could add this to the app submission form on App.co, for sure. But would this also affect a specific App.co API endpoint, and if so, which one? 

And do we know yet if blockstats is interested in pulling (more) data from App.co this way? If not, can we loop them into this conversation?

cc @hstove since he may have thoughts here given his work on the API.

Here's the issue about data ownership you mentioned, for reference: https://github.com/blockstack/app.co/issues/479

**Comment by friedger** [comment]
Yes. (I would name it "make it public data" instead of "porting").

A relevant issue for blockstats is e.g. https://github.com/vrepsys/blockstats-page/issues/4 for the category data. @vrepsys might comment about the likability to resolve that issue :-)

It should be part of meta data for all apis that produce app meta data, like https://app-co-api.herokuapp.com/api/apps, https://app-co-api.herokuapp.com/api/app-mining-apps, etc.
